target/mips/op_helper_cheri.c

```diff
@@ -1023,7 +1042,7 @@ void store_cap_to_memory(CPUArchState *env, uint32_t cs, target_ulong vaddr,
      */
 
     env->statcounters_cap_write++;
-    if (csp->cr_tag) {
+    if (get_capreg_tag_revokable(env,cs)) {
```

Comment:
A minor nit about naming: `get...revokable` makes it sound like it's returning an indication of having *hit* the filter, which makes this line read strangely.  `get_capreg_tag_filtered`?

Reply:
Separately, I see only two calls to `..._revokable()`, so I think you've left out the magic128 case?  Not that I think magic128 is long for the world, but it might be good to keep consistency as far as we can without active testing until we remove it.

Reply:
I think we should remove magic 128. No one uses it and it just causes (albeit minor) maintenance issues.

Reply:
I have no opinions on this. Is it fine for me to not do magic 128 if everyone thinks its dead?

Reply:
Seems fine to me. Magic128 is CHERI256 in 128-bits so not really what we want at all. Different otype width, etc. I think removing it is a good idea. I will do that cleanup the next time I have to change something magic-128 related.

Reply:
Did you not like the suggestion of the new name of `get_capreg_tag_filtered`?